pl_bolts/datamodules/binary_mnist_datamodule.py

```diff
@@ -76,7 +78,7 @@ def __init__(
                 "You want to use transforms loaded from `torchvision` which is not installed yet."
             )
 
-        super().__init__(
+        super().__init__(  # type: ignore[misc]
```

Comment:
mypy will raise errors without these `# type: ignore[misc]` due to the bug reported in https://github.com/python/mypy/issues/6799.